inc/parser/xhtml.php

```diff
@@ -1345,7 +1345,10 @@ function tabletbody_close() {
     function tablerow_open() {
         // initialize the cell counter used for classes
         $this->_counter['cell_counter'] = 0;
-        $class                          = 'row'.$this->_counter['row_counter']++;
+		if (!isset($this->_counter['row_counter'])){
+            $this->_counter['row_counter'] = 0;
```

Comment:
https://github.com/lupo49/dokuwiki-plugin-dir/blob/master/syntax.php#L1066
As #1360 notes, the table_open is redefined. It sets the _counter[row_counter] from this new function, but since that property is protected, it could not be changed by the new open_tabel function.
```php
/** @var array global counter, for table classes etc. */
    protected $_counter = array(); //
```